src/core/light-controller.ts

```diff
@@ -12,11 +12,12 @@ import { localize } from '../localize/localize';
 
 export class LightController extends NotifyBase<LightController> implements ILightContainer {
     private _hass: HomeAssistant;
+    private _lightGroupEntity: LightContainer | undefined;
     private _lights: LightContainer[];
     private _lightsFeatures: LightFeaturesCombined;
     private _defaultColor: Color;
 
-    public constructor(entity_ids: string[], defaultColor: Color) {
+    public constructor(entity_ids: string[], defaultColor: Color, _lightGroupEntityId: string | undefined) {
```

Comment:
Please use `lightGroupEntityId` (without the leading underscore) as argument name.

Reply:
oops, yep missed that thanks